.changeset/modern-coins-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+refactor: Convert Omnichannel helper ee to ts
```

Comment:
refactor usually shouldn't go to a change log.. but if you want to have it there, it is fine =)

Reply:
All right 🤔 didn't know that. We have other refactors with changesets lol

Reply:
changesets are customer facing (or end user) release notes, so that's why **usually** refactors don't deserve a changeset 😉